jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -30,6 +30,7 @@ YUI.add('user-profile', function() {
       dbEnvironmentSet: React.PropTypes.func.isRequired,
       env: React.PropTypes.object.isRequired,
       getDiagramURL: React.PropTypes.func.isRequired,
+      gisf: React.PropTypes.bool.isRequired,
```

Comment:
I see you added an `isRequired`, but I don't see any updates to add this to the tests so I assume there are a bunch of missing prop errors.